File: 4th Sem/OOPJ/Extra Programs/MultiplicationTable.java
import java.util.Scanner;

public class MultiplicationTable
{
    public static void main(String[] args)
    {
        int input;
        Scanner s = new Scanner(System.in);

        System.out.print("Enter Your Number : ");
        input = s.nextInt();

        for(int i=1 ; i<=10 ; i++)
        {
            System.out.print(input + " * " + i + " = " + (i*input));
            System.out.println();
        }
        s.close();
    }
}

File: 3rd Sem/OODP/Extra Programs/MultiplicationTable.cpp
#include<iostream>
using namespace std;

int main()
{
    int num;

    cout << "Enter Your Number: "; cin >> num;

    for (int i = 1; i <= 10; i++)
    {
        cout << "\n";
        cout << "" << num << " * " << i << " = " << num * i;
    }
    cout << "\n";
}